sign/bls/mask.go

```diff
@@ -0,0 +1,172 @@
+package bls
```

Comment:
Shouldn't all of this code go into the new BLS package that supports secure signature aggregation?

Reply:
Moved it to `sign` as suggested offline